server/config.py

```diff
@@ -1,4 +1,8 @@
 import os
+from dotenv import load_dotenv
+load_dotenv()
 
-TEAM_NAME = os.environ['TEAM_NAME']
-MOSHES_GOOGLE_API_KEY = 'AIzaSyAN3q6E8D80h38QQsiSWQ5keW6YSRMPqYc'
+TEAM_NAME = os.getenv('TEAM_NAME')
+MOSHES_GOOGLE_API_KEY = os.getenv('MOSHES_GOOGLE_API_KEY')
+DB_URL = 'postgresql+psycopg2://{user}:{pw}@{url}/{db}'.format(
+    user=os.getenv('DB_USER'), pw=os.getenv('DB_PASSWORD'), url=os.getenv('DB_URL'), db=os.getenv('DB_NAME'))
```

Comment:
Nice that you hid the keys. Just a question. Is there a reason why we cannot just store in .env?

Reply:
the keys are stored in `.env`.  Are you asking why DB_URL is not in `.env`?  thats because string concatenation does not work in a `.env`, so it has to be done in a python file